README.md

```diff
@@ -10,6 +10,25 @@ The Devfile Registry operator manages the lifecycle of the following custom reso
 
 Issue tracking repo: https://github.com/devfile/api with label area/registry
 
+## Requirements
+
+Deployment cluster must meet one of the following criteria:
+
+- OpenShift Container Platform (OCP) 4.12.x
+- Kubernetes 1.25.x-1.26.x
+
+More on the support of container orchestration systems can be 
+found [here](CLUSTER_SUPPORT.md).
+
+Deployments made by the devfile registry operator must *never* target the default namespace due to incompatibility with 
+security setups.
+
+### Development
+
+- Go 1.19.x
+- Docker / Podman
+- Operator SDK 1.28.x
```

Comment:
I wonder if we should tie in the kubernetes version here somehow since the operator sdk supports a specific version.  https://sdk.operatorframework.io/docs/upgrading-sdk-version.  Maybe have a subsection and a list of things to consider when we move up openshift and kube versions?

Reply:
> Maybe have a subsection and a list of things to consider when we move up openshift and kube versions?

I include some references the this guide in the latest commit, I'll add this list of required actions under the **Operator SDK** subsection of `CLUSTER_SUPPORT.md`.